### ISSUE TITLE
segtools-signal-distribution Python 3 compatibility

### ISSUE THREAD

**Issue by paul-sud** [hoffmangroup/segtools#58]
Hello,

I had an old issue on the BitBucket repo asking about Python 3 compatibility for segtools-signal-distribution: https://bitbucket.org/hoffmanlab/segtools/issues/58/segtools-signal-distribution-fails-with . In case that link is down for you too it's archived here: https://web.archive.org/web/20200621152300/https://bitbucket.org/hoffmanlab/segtools/issues/58/segtools-signal-distribution-fails-with

It would be really great if Python 3 would work with this. It looks like you might be able to get away with just removing the Python version checks but from the linked issue it sounded like you wanted to check it yourself.

Best,

Paul

**Comment by paul-sud** [comment]
Hi there, just wondering if you have had a chance to look at this or have any thoughts. There was an old PR for this on Bitbucket but it looks like the repo is down.

**Comment by EricR86** [comment]
Yes, this is near top priority. We don't have an ETA but it's been close to the few things we would like to address sooner than later.

**Comment by paul-sud** [comment]
Glad to hear it, thanks.
